pkg/routeagent_driver/main.go

```diff
@@ -83,6 +84,9 @@ func main() {
 	// set up signals so we handle the first shutdown signal gracefully
 	stopCh := signals.SetupSignalHandler().Done()
 
+	// Clean up "sockets" created as directories by previous versions
+	removeInvalidSockets()
```

Comment:
Q: do we need to run the removeInvalidSockets function every time RA pod is restarted? or we can move it under RA init container code?

Reply:
The init container doesn’t have the required mount volumes — its purpose is to wait until the node state is such that mounts are liable to succeed, so it can’t access anything in the corresponding mounts.